deploy/stacks/container.py

```diff
@@ -309,6 +301,80 @@ def __init__(
             subscriptions_task.task_definition,
         ]
 
+    def create_ecs_security_groups(self, envname, resource_prefix, vpc, vpce_connection, s3_prefix_list, lambdas):
+        scheduled_tasks_sg = ec2.SecurityGroup(
+            self,
+            f'ScheduledTasksSG{envname}',
```

Comment:
It is for scheduled tasks and for cdkproxy tasks right?

Reply:
Yes - originally I had cdkproxy with it's own SG but ultimately I found a way to make cdkproxy ECS tasks work with no traffic over the internet so I removed the additional `cdkproxy_security_group` and now use `scheduled_tasks_sg` for all scheduled tasks and cdkproxy while only shares are handled separately with their own SG